where-for-dinner/spaces-beta3-deployment/aws/routes/k8sGatewayRoutes.yaml

```diff
@@ -4,8 +4,8 @@ metadata:
   name: where-for-dinner-scg-route
   annotations:
    healthcheck.gslb.tanzu.vmware.com/service: where-for-dinner-ui
```

Comment:
Is it intentional that the health check goes to the UI service?

Reply:
good catch!